Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -153,7 +153,7 @@ uplink-buldog-bundle-name = Bulldog Bundle
 uplink-buldog-bundle-desc = Lean and mean: Contains the popular Bulldog Shotgun, a 12g beanbag drum and three 12g buckshot drums.
 
 uplink-grenade-launcher-bundle-name = China-Lake Bundle
-uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 8 rounds of various destruction capability.
+uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 11 rounds of various destruction capability.
```

Comment:
Would something like this work?
```suggestion
uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher. Shells included!
```

Reply:
Yeah, could make the descriptions generic, but it's better players know exactly what they are buying in my opinion. It seems a more permanent solution is wanted than individually fixing descriptions of what contents are inside a bundle.